server/ntb/commands/fix_events_moment_timezone_2023.py

```diff
@@ -0,0 +1,57 @@
+import superdesk
+
+from datetime import datetime, timedelta
+from flask import current_app as app
+
+
+class FixEventsCommand(superdesk.Command):
+    """Fix events affected by moment timezone issue
+
+    which were saved during summer time using the offset of winter time (or vice versa in Australia).
+    """
+
+    option_list = [
+        superdesk.Option("--start", "-s", default="2023-03-26T01:00:00+00:00"),
```

Comment:
One issue we might have, not sure if NTB use this, but the underlying issue is affecting by the specific Event's `timezone`, and if that timezone is inside/outside daylight savings.

This would occur if an Event is created with a different timezone then the system's default timezone.

Reply:
yep they confirmed all events from that period should be affected